tools/cmake/Makefile

```diff
@@ -32,7 +32,12 @@ HOST_CONFIGURE_VARS += \
 HOST_CONFIGURE_ARGS := \
 	$(if $(MAKE_JOBSERVER),--parallel="$(MAKE_JOBSERVER)") \
 	--prefix="$(STAGING_DIR_HOST)" \
-	--generator=Ninja
```

Comment:
we can avoid a minus in the diff, I don't think alphabetical order is important here

Reply:
Done.